src/gtfs/read.rs

```diff
@@ -859,56 +860,68 @@ where
                     transfer.map_err(|e| format_err!("Problem reading {:?}: {}", file, e)),
                     LogLevel::Warn
                 );
-
-                let from_stop_point = skip_error_and_log!(
-                    stop_points
-                        .get(&transfer.from_stop_id)
-                        .ok_or_else(|| format_err!(
-                            "Problem reading {:?}: from_stop_id={:?} not found",
-                            file,
-                            transfer.from_stop_id
-                        )),
+                let expand_stop_area = |stop_id: &str| -> Result<Vec<&StopPoint>> {
+                    if stop_areas.get(stop_id).is_some() {
+                        let list_stop_points = stop_points
+                            .values()
+                            .filter(|stop_point| stop_point.stop_area_id == stop_id)
+                            .collect();
+                        Ok(list_stop_points)
+                    } else {
+                        stop_points
+                            .get(stop_id)
+                            .ok_or_else(|| {
+                                format_err!(
+                                    "Problem reading {:?}: from_stop_id={:?} not found",
+                                    file,
```

Comment:
"Problem reading {:?}: stop_id={:?} not found" seems more appropriate ?

Reply:
Oops :sweat_smile: indeed! Fixed.